.travis.yml

```diff
@@ -350,6 +350,14 @@ jobs:
       script:
         - travis_retry ./scripts/if_changed.sh ./scripts/build.sh $PROJECT $PLATFORM $METHOD
 
+    - stage: test
+      env:
+        - PROJECT=Firestore PLATFORM=macOS METHOD=cmake2
```

Comment:
This doesn't match your addition below.

Also, there's already `PROJECT=Firestore PLATFORM=iOS METHOD=fuzz` that's been cronned below.

If you do want to pursue this, I think we should either:

  * lump this in with another build grouping so that we don't need a new project or method
  * just have a separate method without creating a new project perhaps cmake-fuzz

For the former: would it be possible to build this target along with the ASAN build?